DuckDuckGo/AppDelegate.swift

```diff
@@ -515,18 +515,26 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             presentVPNEarlyAccessOverAlert()
 
             Task {
-                let controller = NetworkProtectionTunnelController()
+                await self.stopAndRemoveVPN(with: "thank-you-dialog")
+            }
+        } else if vpnFeatureVisibility.isPrivacyProLaunched() && !AccountManager().isUserAuthenticated {
```

Comment:
We already set `vpnEarlyAccessOverAlertAlreadyShown` to `true` and return early if `AccountManager().isUserAuthenticated`, but I want to make this check explicit.